bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -163,6 +163,7 @@ spec:
         - mountPath: /run/netns
           name: host-run-netns
           readOnly: true
+          mountPropagation: Bidirectional
```

Comment:
reading the kube docs it seems like this should only need to be `HostToContainer`, not `Bidirectional`

Reply:
alright, I'm going to give that a try